libraries/BLE/examples/central/central.ino

```diff
@@ -0,0 +1,94 @@
+

```

Comment:
The initial version. I will do it after the API related work complete or have  bandwidth.